packages/models-library/src/models_library/api_schemas_webserver/auth.py

```diff
@@ -10,6 +10,28 @@
 class AccountRequestInfo(InputSchema):
     form: dict[str, Any]
 
+    class Config:
+        # NOTE: this is just informative. The format of the form is defined
+        # currently in the front-end and it might change
+        # SEE image in  https://github.com/ITISFoundation/osparc-simcore/pull/5378
+        schema_extra: ClassVar[dict[str, Any]] = {
+            "example": {
+                "form": {
+                    "firstName": "James",
+                    "lastName": "Maxwel",
```

Comment:
Man, everywhere I look in this repo I find this guy 🤔

Reply:
very important guy :-)